ecs-cli/modules/commands/local/local_command.go

```diff
@@ -164,33 +164,33 @@ func flagName(longName string) string {
 func flagDescription(longName, cmdName string) string {
 	m := map[string]map[string]string{
 		flags.TaskDefinitionCompose: {
-			upCmdName: "The Compose file `name` of a task definition to run.",
+			upCmdName: "Specifies the filename `value` that contains the Docker Compose content to run locally.",
 		},
 		flags.TaskDefinitionFile: {
-			createCmdName: fmt.Sprintf("The file `name` of a task definition json to convert. If not specified, defaults to %s", project.LocalInFileName),
-			upCmdName:     fmt.Sprintf("The file `name` of a task definition json to convert and run. If not specified, defaults to %s", project.LocalInFileName),
-			psCmdName:     "List all running containers matching the task definition file `name`.",
-			downCmdName:   "Stop and remove all running containers matching the task definition file `name`.",
+			createCmdName: fmt.Sprintf("Specifies the filename `value` that contains the task definition JSON to convert to a Docker Compose file. If one is not specified, the ECS CLI will look for %s", project.LocalInFileName),
+			upCmdName:     fmt.Sprintf("Specifies the filename `value` containing the task definition JSON to convert and run locally.  If one is not specified, the ECS CLI will look for %s", project.LocalInFileName),
+			psCmdName:     "Lists all running containers matching the task definition filename `value`.",
+			downCmdName:   "Stops and removes all running containers matching the task definition filename `value`.",
 		},
 		flags.TaskDefinitionRemote: {
-			createCmdName: "The `arnOrFamily` of a task definition to convert.",
-			upCmdName:     "The `arnOrFamily` of a task definition to convert and run.",
-			psCmdName:     "List all running containers matching the task definition `arnOrFamily`.",
-			downCmdName:   "Stop and remove all running containers matching the task definition `arnOrFamily`.",
+			createCmdName: "Specifies the full Amazon Resource Name (ARN) or family:revision `value` of the task definition to convert to a Docker Compose file. If you specify a task definition family without a revision, the latest revision is used.",
+			upCmdName:     "Specifies the full Amazon Resource Name (ARN) or family `value` of a task definition to convert and run locally. If you specify a task definition family without a revision, the latest revision is used.",
```

Comment:
nit: Should all of them be "family:revision" `value` instead of just family? Over vice-versa. `createCmdName` has it like that.

Reply:
sure!